### PR TITLE
new lesson postgres insert, update, delete

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,205 @@
...
+
+```SQL
+UPDATE pet
+SET age = 7
+WHERE id = 5;
+```
+
+NOTE: It is very important to ensure the `WHERE` clause is correct, otherwise we might
+update the wrong row!    It is usually a good idea to first write a query using the `SELECT`
+statement with the **before** executing an update to see which rows will be affected:
```
Comment (developerjaym) [inline README.md]: Perhaps remove the words "with the" here

File: README.md [hunk 2]
```diff
@@ -1,32 +1,205 @@
...
+![update age](https://curriculum-content.s3.amazonaws.com/6036/sql-insert-statement/updateage.png)
+
+After every update, select all rows `SELECT * FROM pet`
+and confirm the correct row was updated.  It is ok if your row order
+differs from what is shown below, as long as the values are the same.
+
+![updated row](https://curriculum-content.s3.amazonaws.com/6036/sql-insert-statement/updatedrow.png)
+
+
+**Caution** - If the `UPDATE` statement does not have a `WHERE` clause, all rows will be updated!
```
Comment (developerjaym) [inline README.md:184]: Consider repeating this caution for the DELETE statement, too

Comment (linda-seiter) [reply README.md:184]: good idea.  delete statement updated.

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: fixed typo.

**Review by developerjaym — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,32 +1,222 @@
...
-- Learning Goal 1
-- Learning Goal 2
+- Use an  `INSERT` statement to add a new table row
+- Use an  `UPDATE` statement to change a value in one or more table rows
+- Use a `DELETE` statement to delete one or more table rows
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+SQL provides `INSERT`, `UPDATE`, and `DELETE`  statements to respective
```
Comment (kcleland0818) [inline README.md]: Typo? "respective" --> "respectively"

File: README.md [hunk 4]
```diff
@@ -1,32 +1,222 @@
...
+into a `DELETE` statement.
+
+**Caution** - If the `DELETE` statement does not have a `WHERE` clause, all rows will be deleted!
+
+  
 ## Conclusion
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+We've seen how to use SQL's `INSERT`, `UPDATE`, and `DELETE`  statements to respective
```
Comment (kcleland0818) [inline README.md]: Same typo here

File: README.md (hunk 3 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (linda-seiter) [reply README.md]: fixed

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**